ee/tuf/autoupdate.go

```diff
@@ -247,7 +253,8 @@ func (ta *TufAutoupdater) Execute() (err error) {
 		ta.slogger.Log(context.TODO(), slog.LevelInfo,
 			"checking for updates",
 		)
-		if err := ta.checkForUpdate(context.TODO(), binaries); err != nil {
+		// always allow delay during routine checks for autoupdates
```

Comment:
Here and in comments later (e.g. 345, 407) I would specify "splay delay" or something to distinguish this from "AutoupdateInitialDelay" above